coralproject/talk
# Adds missing template.

## What does this PR do?

Fix this bug: https://www.pivotaltracker.com/story/show/143202269

There was a missing template for the suspension email.

## How do I test this PR?

Report a username
Go to Community page to flagged usernames
Click "Reject"
You will see an option to "Suspend User"
Click "Suspend"
It should work.
